settings.sample.py

```diff
@@ -73,3 +73,9 @@
 # LINTRC_DEFAULTS = './lintrc_defaults.ini'
 
 OK_COMMENT = ':+1: No lint errors found.'
+
+# Set to True to use a label instead of a comment for OK status.
+# Cuts down on github notification noise.
+# Don't forget to create the label on github!
+NO_OK_NOTIFY = False
```

Comment:
The config options should be in the positive form preferrably. Right now its harder to figure out what `NO_OK = False` will end up doing. If instead the config option was `ADD_OK_LABEL` I think the code and behavior would be easier to figure out.